report/generate_report.py

```diff
@@ -371,29 +373,54 @@ def _full_report(project, project_source, date, command, graph_components, raw_f
   </style>
   </head>
   <body>
-  <div class="container-fluid">
-    <div class="row">
-    <div class="col-sm-12">
-      <h1>[<a href="{project_source}">{project}</a>] Report for {date}</h1>
-      </hr>
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-sm-12">
+          <h1>[<a href="{project_source}">{project}</a>] Report for {date}</h1>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col-sm-3 input-group">
+          <span><input id="datePicker" width="150px"/></span>
+          <span><button id="viewReportButton" type="button" class="btn btn-sm btn-link">View Past Report</button><i>(Date & time are in UTC.)</i></span>
+        </div>
+        <script>
+          var reportDate = new Date('{date}');
+          var $datePicker = $('#datePicker').datepicker({{
+              uiLibrary: 'bootstrap4',
+              size: 'small',
+              format: 'yyyy/mm/dd',
+              value: '{date}',
+              disableDates: function (date) {{
+                return date <= reportDate;
```

Comment:
I think we should change this to disable dates that are really in the future, not in the future of this report.

Reply:
Good catch! Fixed.